contracts/services/rebalancer/src/rebalance.rs

```diff
@@ -668,6 +688,7 @@ fn generate_trades_msgs(
                 if let Ok(msg) = construct_msg(deps, auction_manager.clone(), trade.clone()) {
                     max_sell -= token_sell.auction_min_send_value;
                     msgs.push(msg);
+                    trades.push(trade);
```

Comment:
Is this coming from another PR ?

Reply:
Its a small bug I noticed, we were missing trades in our events for when a `min_balance` limit was hit.